apps/web/netlify.toml

```diff
@@ -1,3 +1,13 @@
+[build.environment]
+  DISABLE_ESLINT_PLUGIN="true"
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This disables linting when performing Netlify builds -- this was built-in to `react-app-rewired`. 

In general, I don't believe Netlify should be concerned with linting, and it should be done in a separate, modular CI action. For this specific PR though, it was required because it _lints the entire package‏_, which will fail with the new rule.

Reply:
I agree 100%. Linting is a local, pre-commit concern.